tensorflow/lite/micro/kernels/softmax_test.cc

```diff
@@ -322,4 +322,360 @@ TF_LITE_MICRO_TEST(SimpleTestQuantizedSigned) {
       output_data);
 }
 
+TF_LITE_MICRO_TEST(SimpleTestQuantizedSigned1D) {
+  using tflite::testing::F2QS;
+
+  const float input_min = -63.5f;
+  const float input_max = 64.0f;
+  const float output_min = 0.0f;
+  const float output_max = (255.0f / 256.0f);
+  const int output_dims_count = 5;
+  int8_t output_data[output_dims_count];
+  tflite::testing::TestSoftmaxQuantizedSigned(  //
+      {1, 5},                                   // Input shape.
+      {
+          F2QS(1.0, input_min, input_max),
+          F2QS(2.0, input_min, input_max),
+          F2QS(3.0, input_min, input_max),
+          F2QS(4.0, input_min, input_max),
+          F2QS(5.0, input_min, input_max),
+      },
+      input_min, input_max,  // Input quantized range.
+      {
+          // Expected results.
+          F2QS(0.011656231, output_min, output_max),
+          F2QS(0.031684921, output_min, output_max),
+          F2QS(0.086128544, output_min, output_max),
+          F2QS(0.234121657, output_min, output_max),
+          F2QS(0.636408647, output_min, output_max),
+      },
+      {1, 5},                  // Output shape.
+      output_min, output_max,  // Output quantized range.
+      output_data);
+}
+
+TF_LITE_MICRO_TEST(SimpleTestQuantizedSigned2D) {
+  using tflite::testing::F2QS;
+
+  const float input_min = -63.5f;
+  const float input_max = 64.0f;
+  const float output_min = 0.0f;
+  const float output_max = (255.0f / 256.0f);
+  const int output_dims_count = 5;
+  int8_t output_data[output_dims_count];
+  tflite::testing::TestSoftmaxQuantizedSigned(  //
```

Comment:
`==1387==ERROR: AddressSanitizer: stack-buffer-overflow on address  at pc  bp  sp 
WRITE of size 1 at  thread T0
    #0  in void tflite::reference_integer_ops::Softmax<signed char>(tflite::SoftmaxParams const&, tflite::RuntimeShape const&, signed char const*, tflite::RuntimeShape const&, signed char*) third_party/tensorflow/lite/kernels/internal/reference/integer_ops/softmax.h:93:36
    #1  in tflite::ops::micro::activations::Softmax2DQuantized(TfLiteTensor const*, TfLiteTensor*, TfLiteSoftmaxParams*, tflite::ops::micro::activations::(anonymous namespace)::OpData*) third_party/tensorflow/lite/micro/kernels/softmax.cc:154:7
    #2  in tflite::ops::micro::activations::SoftmaxEval(TfLiteContext*, TfLiteNode*) third_party/tensorflow/lite/micro/kernels/softmax.cc:233:9
    #3  in tflite::testing::(anonymous namespace)::TestSoftmaxQuantizedSigned(std::initializer_list<int>, std::initializer_list<signed char>, float, float, std::initializer_list<signed char>, std::initializer_list<int>, float, float, signed char*) third_party/tensorflow/lite/micro/kernels/softmax_test.cc:212:3
    #4  in main third_party/tensorflow/lite/micro/kernels/softmax_test.cc:366:3
    #5  in __libc_start_main (/usr/grte/v4/lib64/libc.so.6+0x38bbc)
    #6  in _start /usr/grte/v4/debug-src/src/csu/../sysdeps/x86_64/start.S:108

Address  is located in stack of thread T0 at offset 565 in frame
    #0  in main third_party/tensorflow/lite/micro/kernels/softmax_test.cc:225
`